src/stories/ReturnDateFiel.stories.js

```diff
@@ -0,0 +1,8 @@
+import React from "react";
+import ReturnDate from "../components/ReturnDate";
+
+export default {
+  title: "ReturnDate"
+};
+
+export const ReturnDateField = () => <ReturnDate></ReturnDate>;
```

Comment:
The filename is missing a "d", and you could have these two components consolidated in one component, like `SelectTripPeriod` or something and then just have one story for that. `StartDateField` and `ReturnDateField` will probably always used together, so might as well put them in one component

Reply:
Thank you Marwin,